ccViewer/Mac/CMakeLists.txt

```diff
@@ -1,12 +1,24 @@
-# Put together our App bundle on Mac OS X
+# Put together our App bundle on macOS
 if( APPLE )
-   # this will install all the Qt libs and the necessary plugins
-   install( CODE "execute_process( COMMAND \"${MACDEPLOYQT}\" \"${CCVIEWER_MAC_BASE_DIR}\" -verbose=1 )" COMPONENT Runtime )
+	set_target_properties( ${PROJECT_NAME} PROPERTIES
+		INSTALL_RPATH "@executable_path/../Frameworks"
+		MACOSX_BUNDLE_INFO_PLIST ${CMAKE_CURRENT_SOURCE_DIR}/ccViewer.plist )
 
-   # install icons
-   install( FILES ../images/icon/cc_viewer_icon.icns DESTINATION ${CCVIEWER_MAC_BASE_DIR}/Contents/Resources COMPONENT Runtime )
+	set( MACOSX_BUNDLE_ICON_FILE cc_viewer_icon.icns )
+	set( MACOSX_BUNDLE_SHORT_VERSION_STRING "${PROJECT_VERSION}" )
+	set( MACOSX_BUNDLE_LONG_VERSION_STRING "${PROJECT_VERSION}" )
+	set( MACOSX_BUNDLE_BUNDLE_VERSION "${PROJECT_VERSION}" )
 
-   # remove any dev-machine specific lib paths
-   install( CODE "execute_process( COMMAND \"${CMAKE_SOURCE_DIR}/scripts/mac/fix_libs.sh\"
```

Comment:
CGAL is optionally header only starting from 4.9 and it's the default since 5.0. Maybe you have to check the version of CGAL and force/check if header only is enabled. What do you think?

Reply:
This was actually a hack-workaround for homebrew that was used for building releases. (And the script had a hard-coded path & version numbers in it.)

I don't think the complexity of checking versions and keeping this is worth it.